src/query.js

```diff
@@ -6,13 +6,17 @@ export const Reducible = type(class Reducible {
     return reduce(reducible, fn, initial);
   }
 
-  sum(reducible, getter) {
-    return reduce(reducible, (total, member) => total + getter(member), 0);
-  }
-
   filter(reducible, predicate) {
     return reduce(reducible, (filtered, member) => predicate(member) ? filtered.concat(member) : filtered, []);
   }
 });
 
-export const { reduce, sum, filter } = Reducible.prototype;
+export const Mappable = type(class Mappable {
```

Comment:
How is this different than Functor?

Reply:
Functor should return the same type as the original functor. Mappable maps to array. In context of React, Functor instance can not be mapped to components.